pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
I don't know if it's not allowed (the error page doesn't show it: https://symfony.sh/r/contrib/9) but it's sure that it will be ignored until the package is moved to symfony/recipes.